packages/redux/src/sharedWishlists/selectors.ts

```diff
@@ -0,0 +1,197 @@
+import * as fromSharedWishlistReducer from './reducer';
+import { createSelector } from 'reselect';
+import { getEntityById } from '../entities/selectors';
+import { getProduct } from '../products/selectors/product';
+import type {
+  SharedWishlist,
+  SharedWishlistItem,
+} from '@farfetch/blackout-client';
+import type {
+  SharedWishlistEntity,
+  SharedWishlistItemEntity,
+  SharedWishlistItemHydrated,
+} from '../entities/types';
+import type { SharedWishlistState } from './types';
+import type { StoreState } from '../types';
+
+/**
+ * Retrieves current user's shared wishlist id.
+ *
+ * @example
+ * ```
+ * import { getSharedWishlistId } from '@farfetch/blackout-redux/sharedWishlists';
```

Comment:
We need to use imports from the root of the package like this:

```suggestion
 * import { getSharedWishlistId } from '@farfetch/blackout-redux';
```

Reply:
Fixed